_Frontmatter/announcement.md

```diff
@@ -4,8 +4,11 @@ title: Announcement
 navOrder: 3
 navTitle: Announcement
 permalink: /announcement/
+anchor: announcement
 ---
 
+[](announcement.md#announcement){:name="announcement"}
+
```

Comment:
```suggestion

# ANNOUNCEMENT

```

Reply:
Doing it this way would break the PDF because it required special formatting, but we can add a template for this page to add the header around the page (this is how the PDF handles it).